docs/src/pages/[platform]/connected-components/storage/fileuploader/react.mdx

```diff
@@ -168,6 +168,26 @@ You can limit what users upload with these 3 props:
   </ExampleCode>
 </Example>
 
+## Setting a Bucket
+
+If you have configured your Amplify project to use multiple S3 buckets, you can use the `bucket` prop to choose which of the buckets the component will use:
```

Comment:
Can we link to Multibucket setup in main docs from here?

Reply:
Good call. Added a link